js/scripts.js

```diff
@@ -0,0 +1,28 @@
+$(function() {
+    var url = 'https://restcountries.eu/rest/v1/name/';
+    var countriesList = $('#countries');
+
+    $('#search').click(searchCountries);
+    $('#country-name').focus(clearImput);
+
+    function searchCountries() {        
+        
+        var countryName = $('#country-name').val();
+        if (!countryName.length) countryName = 'Poland';
+        $.ajax({
+            url: url + countryName,
+            method: 'GET',
+            success: showCountriesList
+        });
+    }
+
+    function showCountriesList(resp) {
+        countriesList.empty();
```

Comment:
before you do something with your response you should check that data in response contain same model of data that you expect